policies/stable-release-updates.md

```diff
@@ -6,8 +6,10 @@ This policy covers allowed changes on stable release branches.
 Definitions
 -----------
 
-A **stable release** is a patch release from an existing supported minor
-release branch.
+A **stable release** is a series including a major or minor release
```

Comment:
_beginning with_ instead of _including_?

Something feels wrong here but I'm not able to put my finger on it.

Reply:
Sure, seems like a good change